cirq/contrib/qasm_import/_parser.py

```diff
@@ -331,10 +339,30 @@ def p_arg_bit(self, p):
             if arg_name not in self.qubits.keys():
                 self.qubits[arg_name] = NamedQubit(arg_name)
             p[0] = [self.qubits[arg_name]]
+        elif reg in self.cregs.keys():
+            p[0] = [arg_name]
         else:
             raise QasmException('Undefined quantum register "{}" '
                                 'at line {}'.format(reg, p.lineno(1)))
 
+    # measurement operations
```

Comment:
Does anything check that the type of register is correct? E.g. that you don't measure classical into quantum.

Reply:
yeah that check was missing - solved it by handling quantum and classical registers separately